### ISSUE TITLE
Game crashed using Red Key

### ISSUE THREAD

**Issue by Godavarism** [UnderscoreKilburn/antibirth-bugs#26]
Normally the Red Key works very smoothly for me but for some reason this time it crashed and gave me this error code:

![](http://puu.sh/tPENR/391431d2bf.png)

Details: I was doing the "When Life Gives you Lemons" challenge, the seed was P9LK-9HF8, and the crash happened in Utero 1 after going down two rooms from the spawn (to the combat room) and trying to open the red door to my right. I can reproduce this crash consistently by reloading the save.

**Comment by Godavarism** [comment]
Addendum: This actually happens every time I try to open a red door, regardless of its position.

**Comment by AnonTheMouse** [comment]
Were you, by any chance, continuing after crashing or closing out of the game?

**Comment by Godavarism** [comment]
Yes, I don't have any specific error code but I had one of the "normal"
crashes which seem to happen occasionally and had to restart the floor.

**Comment by UnderscoreKilburn** [comment]
Ah that's a known issue then, there are a bunch of different factors causing it and I'm currently looking into fixing most of them.
